### PR TITLE
Add script to check SDL does not resend OnSeekMediaClockTimer if seek…

### PR DESCRIPTION
**Pull request by OVikhrov**
…Time hours out upper bound
@aarotar please review

### CODE REVIEW

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Out_Upper_Bound.lua [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+-- b) OnSeekMediaClockTimer notification allowed in preloaded file for default app
+-- c) App successfylly registered, consented and activated
+--
+-- 2. Performed steps:
+-- a) HMI sends OnSeekMediaClockTimer notification to SDL with hours out upper bound value ("60") of seekTime parameter.
+--
+-- Expected result:
+-- a) SDL does not resend OnSeekMediaClockTimer notification to mobile app.
+---------------------------------------------------------------------------------------------
+
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Out_Upper_Bound.lua]: @OVikhrov, plase avoid redundant empty lines

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Out_Upper_Bound.lua [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+function Test:Preconditions_ActivateApp()
+commonSteps:ActivateAppInSpecificLevel(self, self.applications["Test Application"])
+EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_OnSeekMediaClockTimer_hours_Value_Of_seekTime_Is_Out_Upper_Bound()
+ self.hmiConnection:SendNotification("UI.OnSeekMediaClockTimer",{seekTime =  {hours = 60, minutes = 3, seconds = 3}, appID = self.applications["Test Application"]})
+
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Out_Upper_Bound.lua]: @OVikhrov, plase avoid redundant empty lines

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Out_Upper_Bound.lua (hunk 1 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Out_Upper_Bound.lua]: Accepted

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Out_Upper_Bound.lua (hunk 2 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_hours_Out_Upper_Bound.lua]: Accepted

**Comment by OVikhrov** [comment]
@aarotar Plese review iteration#2

**Pull request merged**